src/Makefile.am

```diff
@@ -12,6 +12,7 @@ AM_CPPFLAGS = \
 	-DRUN_DIR="\"$(localstatedir)/run/netdata\"" \
 	-DWEB_DIR="\"$(webdir)\"" \
 	$(NULL)
+
 AM_CFLAGS = \
```

Comment:
You are right.
But there are cflags that should be applied only to specific programs.

For example, libcap only at apps.plugin, libipmimonitoring to freeipmi.plugin, etc

 How can I do this?